frontend/wallet/src/render/contexts/IntlProvider.re

```diff
@@ -0,0 +1,25 @@
+type action =
+  | SetLocale(Locale.locale);
+
+let initialState = Locale.Vn;
+
+let intlReducer = (_, action) =>
+  switch (action) {
+  | SetLocale(locale) => locale
+  };
+
+module IntlContextType = {
+  type t = (Locale.locale, action => unit);
+
+  let initialContext = (Locale.En, _ => ());
+};
+
+// TODO Somehow insert ReactIntl.Provider here...
+
+type t = IntlContextType.t;
```

Comment:
@Schmavery How do I actually wrap ReactIntl.Provider in your current setup with ContextProvider.Make?

Reply:
I am still struggling on how to create a wrap the ReactIntl.Provider with my own custom provider.

Reply:
If it provides its own provider component you should be able to just use that, not sure why you need to wrap it (maybe I'm missing sometihng).
The ContextProvider is just a way to make a provider for some context value.